sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -237,6 +249,13 @@ def start_k8s_software(self):
         """)
         self.host_node.remoter.run(f"sudo -E bash -cxe '{script}'")
 
+    def on_deploy_completed(self):
+        docker_repo = self.params.get('docker_image')
+        scylla_version = self.params.get('scylla_version')
+        if not scylla_version or not docker_repo:
+            return
+        self.docker_pull(f"{docker_repo}:{scylla_version}")
```

Comment:
Please, define this part in one place and reuse it in "kind" adding it's additional line after calling main func.
Or in short, please, remove code duplication.

Reply:
done